spaces/S000123/properties/P000045.md

```diff
@@ -7,5 +7,4 @@ refs:
   name: Counterexamples in Topology
 ---
 
-Asserted in the General Reference Chart for space #126 in
-{{doi:10.1007/978-1-4612-6290-9_6}}.
+See item #3 for space #126 in {{doi:10.1007/978-1-4612-6290-9_6}}.
```

Comment:
```suggestion
$X$ is {P36} by item #2 for space #126 in {{doi:10.1007/978-1-4612-6290-9_6}}.
And the point $\omega$ is a dispersion point of $X$, since $X\setminus\{\omega\}$ is {S124} and {S124|P47}.
```